src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -269,19 +269,24 @@ private boolean setUpLogging(ContestConfig config) {
     // At each iteration of the following loop, we add records from another source file.
     for (RawContestConfig.CvrSource source : config.rawConfig.cvrFileSources) {
       String cvrPath = config.resolveConfigPath(source.getFilePath());
+      Provider provider = ContestConfig.getProvider(source);
       try {
         if (ContestConfig.isCdf(source)) {
           Logger.log(Level.INFO, "Reading CDF cast vote record file: %s...", cvrPath);
           CommonDataFormatReader reader = new CommonDataFormatReader(cvrPath, config);
           reader.parseCvrFile(castVoteRecords);
           continue;
-        } else if (ContestConfig.getProvider(source) == Provider.ESS) {
+        } else if (provider == Provider.DOMINION) {
+          Logger.log(Level.INFO, "Reading Dominion cast vote records from folder: %s...", cvrPath);
+          new DominionCvrReader(cvrPath).readCastVoteRecords(castVoteRecords);
```

Comment:
So this seems to work great, but I don't think it's creating the intermediary CVR .csv file (e.g. like `dominion_cvr_conversion_alaska_contest_1_expected.csv`). Wasn't this one of our requirements? I imagine we'd want it to appear in the output folder too.